typescript/src/languages/en-US/settings.json

```diff
@@ -19,10 +19,16 @@
 	"channelsLogsEmojiDelete": "The channel for emoji deletion logs, if set, I will send a message when an emoji is deleted.",
 	"channelsLogsEmojiUpdate": "The channel for emoji update logs, if set, I send a message when an emoji is updated in any way. This message will contain the changes made to the emoji.",
 	"channelsLogsImage": "The channel I will use to re-upload all images I see.",
-	"channelsLogsMember": "The channel for member logs, you must enable the events (`events.member-add` for new members, `events.member-remove` for members who left).",
-	"channelsLogsMessage": "The channel for (non-NSFW) message logs, you must enable the events (`events.message-delete` for deleted messages, `events.message-edit` for edited messages).",
+	"channelsLogsMemberAdd": "The channel I will use to notify when a member joins.",
+	"channelsLogsMemberRemove": "The channel I will use to notify when a member leaves.",
+	"channelsLogsMemberNickNameUpdate": "The channel I will use to notify when a member changes their nickname.",
+	"channelsLogsMemberUserNameUpdate": "The channel I will use to notify when a member changes their username.",
+	"channelsLogsMemberRoleUpdate": "The channel I will use to notify when a member's roles have been changed.",
+	"channelsLogsMessageDelete": "The channel I will use to notify when a message has been deleted.",
+	"channelsLogsMessageDeleteNsfw": "The channel I will use to notify when a message has been deleted, if sent in a NSFW channel.",
+	"channelsLogsMessageUpdate": "The channel I will use to notify when a message has been updated.",
+	"channelsLogsMessageUpdateNsfw": "The channel I will use to notify when a message has been updated, if sent in a NSFW channel.",
```

Comment:
```suggestion
	"channelsLogsMemberAdd": "The channel I will send a message to when a member joins.",
	"channelsLogsMemberRemove": "The channel I will send a message to  when a member leaves, is kicked, or is banned.",
	"channelsLogsMemberNickNameUpdate": "The channel I will send a message to when a member changes their nickname.",
	"channelsLogsMemberUserNameUpdate": "The channel I will send a message to when a member changes their username.",
	"channelsLogsMemberRoleUpdate": "The channel I will send a message to when a member gets or loses a role.",
	"channelsLogsMessageDelete": "The channel I will send a message to when a message has been deleted.",
	"channelsLogsMessageDeleteNsfw": "The channel I will send a message to when a message from an NSFW channel has been deleted.",
	"channelsLogsMessageUpdate": "The channel I will send a message to when a message has been updated.",
	"channelsLogsMessageUpdateNsfw": "The channel I will send a message to when a message has been updated, if sent in a NSFW channel.",
```

btw it's `an NSFW` because you pronounce `NSFW` as `En Es Ef Double You`, the first letter here is a vowel sound. Similarly it's also `an NFL player`:
> [Tom] Brady has started 344 games (299 regular season, 45 playoff) in 21 seasons, the most for an NFL quarterback.

Reply:
I always pronounced `NSFW` as `Not Safe For Work` lmao